components/dashboard/src/projects/new-project/NewProjectCreateFromURL.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC, useCallback, useMemo } from "react";
+import isURL from "validator/lib/isURL";
+import { CreateProjectArgs } from "../../data/projects/create-project-mutation";
+import { Subheading } from "../../components/typography/headings";
+import { Button } from "../../components/Button";
+import { useToast } from "../../components/toasts/Toasts";
+
+type Props = {
+    repoSearchFilter: string;
+    isCreating: boolean;
+    onCreateProject: (args: CreateProjectArgs) => void;
+};
+export const NewProjectCreateFromURL: FC<Props> = ({ repoSearchFilter, isCreating, onCreateProject }) => {
+    const { toast } = useToast();
+    const showCreateFromURL = useMemo(() => {
+        // TODO: Only accounts for https urls, need to account for ssh clone urls too?
+        const looksLikeURL = isURL(repoSearchFilter, {
+            require_protocol: true,
+            protocols: ["https"],
+        });
+
+        const hasTrailingGit = /\.git$/.test(repoSearchFilter);
```

Comment:
This is also true when the URL looks like this:
```
https://example.com#.git
https://example.com?s=.git
```

It's a pretty small issue and the project creation will fail anyway, but wanted to bring it up.